src/stores/clusterlabels/reducer.ts

```diff
@@ -0,0 +1,36 @@
+import produce from 'immer';
+
+import { updateClusterLabels } from './actions';
+
+interface IState {
+  requestInProgress: boolean;
+  error: Error | null;
+}
+
+const initialState: IState = {
+  requestInProgress: false,
+  error: null,
+};
+
+const clusterLabelsReducer = produce((draft: IState, action) => {
+  const { types } = updateClusterLabels();
+  switch (action.type) {
+    case types.request:
+      draft.requestInProgress = true;
+      draft.error = null;
+
+      return;
```

Comment:
```suggestion
      break;
```

Reply:
`return` vs `break` here: Does this change anything since the other cases don't get executed either way?

Reply:
It does not change the behaviour of the code. It's just a matter of using the correct syntax for the `switch` statement
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch